training/pytorch/structured/custom_containers/gpu/README.md

```diff
@@ -0,0 +1,90 @@
+[metadata.py](trainer/metadata.py)     | Defines: 1) task type, 2) input data header, 3) numeric and categorical feature names, and 4) target feature name (and labels, for a classification task)                                                                                                                                                                              | **Yes**, as you will need to specify the metadata of your dataset. **This might be the only module to change!**
+[inputs.py](trainer/inputs.py)         | Includes: 1) data input functions to read data from csv files, 2) parsing functions to convert csv to tensors, 3) function to implement your custom features processing and creation functionality, and 4) prediction functions (for serving the model) that accepts CSV, JSON, and tf.example instances. | **Maybe**, if you want to implement any custom pre-processing and feature creation during reading data.
+[model.py](trainer/model.py)           | Includes: 1) function to create DNNLinearCombinedRegressor, and 2) DNNLinearCombinedClassifier.                                                                                                                                                                                                                                        | **Yes** you want to cutomize the model to your inputs, the loss function, and the optimizer function.
+[experiment.py](trainer/experiment.py)       | Runs the model training and evaluation experiment, and exports the final model.                                                                                                                                                                                                                                                        | **No, unless** you want to add/remove parameters, or change parameter default values.
+[task.py](trainer/task.py)             | Includes: 1) Initialise and parse task arguments (hyper parameters), and 2) Entry point to the trainer.                                                                                                                                                                                                                                | **No, unless** you want to add/remove parameters, or change parameter default values.
+
+### Scripts
+
+* [train-local.sh](scripts/train-local) This script trains a model locally. 
+  It generates a SavedModel in local folder on the Docker Image.
+
+* [train-cloud.sh](scripts/train-cloud.sh) This script submits a training job to AI Platform.
+
+## How to run
+
+Once the prerequisites are satisfied, you may:
+
+1. For local testing, run: (Note: if you don't have a GPU, the program will be run on the CPU)
+    ```
+    source ./scripts/train-local.sh
+    ```
+2. For cloud testing, run:
```

Comment:
For GPU in cloud same YAML configuration to select # of GPU and GPU model?

Reply:
I change the cloud script to be a `--scale-tier BASIC_GPU`, should I make it a yaml file instead?